src/components/EditTodo/EditTodo.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { saveChangeTask } from 'components/Service/service';
+import 'components/Todos/style.scss'
+
+const EditTodo = ({allTasks, text, _id, setbuttonEditTask, setAllTasks}) => {
+  const [valueChangeInput, setValueChangeInput] = useState(text);
+
+  const handleKey = (e) => {
+    if (e.key === "Enter") {
+      saveTask();
+    };
+  };
+
+  const isValidChangeInputText = (valueChangeInput) => {
```

Comment:
If your validation is repeated in two components, can it be moved to helpers ? And import from there?